Provider/src/Startup.cs

```diff
@@ -1,39 +1,24 @@
-﻿using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
-using provider.Repositories;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Provider.Repositories;
 
-namespace Provider
-{
-    public class Startup
-    {
-        public Startup(IConfiguration configuration)
-        {
-            Configuration = configuration;
-        }
-
-        public IConfiguration Configuration { get; }
-
-        // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddSingleton<IProductRepository, ProductRepository>();
-            services.AddControllers()
-                        .AddJsonOptions(options => options.JsonSerializerOptions.WriteIndented = true);
-            services.AddMvc();
-        }
-
-        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
-        {
-            if (env.IsDevelopment())
-            {
-                app.UseDeveloperExceptionPage();
-            }
-            app.UseRouting();
-            app.UseEndpoints(e => e.MapControllers());
-        }
-    }
-}
+namespace Provider;
+
+public static class Startup
```

Comment:
Is the startup file still required here with the MinimalApi? We have eliminated all Startup files for .Net 6.0 in our projects, and struggling where to define the Pact Verifier Middleware in the new solution without a Startup.